doc/source/dask-on-ray.rst

```diff
@@ -25,49 +31,88 @@ Here's an example:
 
-   @dask.delayed
-   def add(x, y):
-       time.sleep(3)
-       return x + y
+   # Set the scheduler to ray_dask_get in your config so you don't have to specify it on
+   # each compute call.
+   dask.config.set(scheduler=ray_dask_get)
+
+   df = dd.from_pandas(pd.DataFrame(np.random.randint(0, 100, size=(1024, 2)), columns=["age", "grade"]))
+   df.groupby(["age"]).mean().compute()
 
-   x = inc(1)
-   y = inc(2)
-   z = add(x, y)
-   # The Dask scheduler submits the underlying task graph to Ray.
-   z.compute(scheduler=ray_dask_get)
+
+.. note::
+  For execution on a Ray cluster, you should *not* use the
+  `Dask.distributed <https://distributed.dask.org/en/latest/quickstart.html>`__
+  client; simply use plain Dask and its collections, and pass ``ray_dask_get``
+  to ``.compute()`` calls or set the scheduler in one of the other ways detailed `here <https://docs.dask.org/en/latest/scheduling.html#configuration>`__. Follow the instructions for
+  :ref:`using Ray on a cluster <using-ray-on-a-cluster>` to modify the
+  ``ray.init()`` call.
 
 Why use Dask on Ray?
 
-   1. If you'd like to create data analyses using the familiar NumPy and Pandas
-      APIs provided by Dask and execute them on a production-ready distributed
-      task execution system like Ray.
-   2. If you'd like to use Dask and Ray libraries in the same application
-      without having two different task execution backends.
+   1. If you'd like to create data analyses using the familiar NumPy and Pandas APIs provided by Dask and execute them on a fast, fault-tolerant distributed task execution system geared towards production, like Ray.
+   2. If you'd like to use Dask and Ray libraries in the same application without having two different clusters.
    3. To take advantage of Ray-specific features such as the
       :ref:`cluster launcher <ref-automatic-cluster>` and
       :ref:`shared-memory store <memory>`.
 
-Note that for execution on a Ray cluster, you should *not* use the
-`Dask.distributed <https://distributed.dask.org/en/latest/quickstart.html>`__
-client; simply use plain Dask and its collections, and pass ``ray_dask_get``
-to ``.compute()`` calls. Follow the instructions for
-:ref:`using Ray on a cluster <using-ray-on-a-cluster>` to modify the
-``ray.init()`` call.
+Dask-on-Ray is an ongoing project and is not expected to achieve the same performance as using Ray directly. All `Dask abstractions <https://docs.dask.org/en/latest/user-interfaces.html>`__ should run seamlessly on top of Ray using this scheduler, so if you find that one of these abstractions doesn't run on Ray, please `open an issue <https://github.com/ray-project/ray/issues/new/choose>`__.
+
+================================================
+Out-of-Core Data Processing
+================================================
+
+Processing datasets larger than cluster memory is supported via Ray's `object spilling
+<https://docs.ray.io/en/master/memory-management.html#object-spilling>`__: if
```

Comment:
We shouldn't tag master here directly. There's a way to make a link natively. (Use this as an example; https://github.com/ray-project/ray/blob/master/doc/examples/placement-group.rst)